src/main/java/com/autotune/experimentManager/handler/PreValidationHandler.java

```diff
@@ -53,7 +56,17 @@ public void execute(ExperimentTrial experimentTrial, TrialDetails trialDetails,
             stepsMeatData.setBeginTimestamp(new Timestamp(System.currentTimeMillis()));
             /**
              * Implement Prevalidation Logic
+             *
+             * Adding the input JSON check for query validity
              */
+            MetricsValidator metricsValidator = Validator.getMetricsValidator();
+            CommonUtils.QueryValidity validMetrics = metricsValidator.validateTrialMetrics(experimentTrial);
+            if (CommonUtils.QueryValidity.VALID != validMetrics) {
+                /**
+                 * Need to stop the experiment.
+                 */
+                System.out.println("InValid metrics");
```

Comment:
Please use LOGGER here

Reply:
Made changes accordingly, exact failure log will be updated to `resulterror` in the upcoming PR's